scripts/bootstrap.sh

```
@@ -0,0 +1,56 @@
+az group create --name "$RG_NAME" --location "$LOCATION"
+
+echo "Creating Storage Account: $SA_NAME..."
+az storage account create --resource-group "$RG_NAME" --name "$SA_NAME" --sku Standard_LRS --encryption-services blob
+
+echo "Creating Storage Container: $CONTAINER_NAME..."
+az storage container create --name "$CONTAINER_NAME" --account-name "$SA_NAME"
+
+# --- Create Azure AD Application and Service Principal for GitHub Actions OIDC ---
+echo "Creating Azure AD Application: $APP_NAME..."
+APP_ID=$(az ad app create --display-name "$APP_NAME" --query appId --output tsv)
+SP_ID=$(az ad sp create-for-rbac --name "$APP_NAME" --role "Contributor" --scopes "/subscriptions/$(az account show --query id --output tsv)" --query appId --output tsv)
```

Shellcheck (0.11.0)

[warning] SP_ID appears unused. Verify use (or export if used externally).

(SC2034)